test/TestProjectWithMasterReference/Procedures/csp_MyProcedureThatUsesMaster.sql

```diff
@@ -0,0 +1,6 @@
+CREATE PROCEDURE [dbo].[csp_MyProcedureThatUsesMaster]
+    @p_SomeParam int
```

Comment:
nit: this parameter isn't needed. Doesn't hurt anything either, so no big deal, just thought I'd point it out!

Reply:
Sure, I've removed it :).